CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 # [3.0.2](https://github.com/phalcon/cphalcon/releases/tag/v3.0.2) (2016-XX-XX)
+- Fixed `Phalcon\Assets\Manager` joining resources without filter [#12370](https://github.com/phalcon/cphalcon/issues/12370)
```

Comment:
This will generate conflicts when rebase every time for each developer. Could you please add this to the end of 3.0.2 stack?